pay-api/src/pay_api/models/eft_credit_invoice_link.py

```diff
@@ -62,3 +64,8 @@ def find_pending_invoice_links(cls, invoice_id: int):
                 .filter_by(invoice_id=invoice_id)
                 .filter(cls.status_code.in_([EFTCreditInvoiceStatus.PENDING.value]))
                 ).one_or_none()
+
+    @classmethod
+    def get_next_group_link_seq(cls):
+        """Get next value of EFT Group Link Sequence."""
+        return db.session.execute(text("SELECT nextval('eft_group_link_seq')")).scalar()
```

Comment:
I'm guessing this auto increments the sequence

Reply:
We need to manually increment it as I want to use it for a batch of records (multiple inserts).